resources/definitions/fdmprinter.def.json

```diff
@@ -3379,6 +3379,30 @@
                     "settable_per_mesh": false,
                     "settable_per_extruder": true
                 },
+                "support_skip_some_zags":
+                {
+                    "label": "Skip Some ZigZags Connections",
+                    "description": "Skip some ZigZags connections to make the support structure easier to break.",
+                    "type": "bool",
+                    "default_value": false,
+                    "enabled": "support_enable and (support_pattern == 'zigzag')",
+                    "limit_to_extruder": "support_infill_extruder_nr",
+                    "settable_per_mesh": false,
+                    "settable_per_extruder": true
+                },
+                "support_zag_skip_count":
+                {
+                    "label": "ZigZag Connection Skip Count",
```

Comment:
rename to 'Support Line Connection Skip Cycle'

'Skin Count' suggests that this setting governs the amount of connections skipped, while it is about 1 in how many connections to skip.

Perhaps not 'skip cycle', but 'skip period' or 'skip periodicity'. Please discuss with Ruben.